src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -812,8 +831,149 @@ def __set_errors_json(self, error_count_by_operation, errors_by_operation):
             "details": errors_by_operation,
             "message": message
         }
+
+    def __set_error_detail(self, error_code, message):
+        formatted_message = self.__ensure_error_message_restriction_compliance(message)
+        return {
+            "code": str(error_code),
+            "message": str(formatted_message)
+        }
     # endregion
 
+    # region - Patch Truncation
+    def get_num_assessment_patches_removed(self):
+        return len(self.__assessment_patches_removed)
+
+    def get_num_installation_patches_removed(self):
+        return len(self.__installation_patches_removed)
+
+    def log_truncated_patches(self):
+        """ log details of all the removed patches from status """
+        self.composite_logger.log_debug("Count of patches removed from: [Assessment={0}] [Installation={1}]".format(self.get_num_assessment_patches_removed(), self.get_num_installation_patches_removed()))
```

Comment:
This can remain log_debug. (Marking an exception to the log_verbose ask)

Reply:
leaving it log_debug,
done